### PR TITLE
Add parameter comment in timestamp API

### PR DESCRIPTION
**Pull request by djKooks**
I add parameter comment in timestamp API. I commented to make a input as string.
Please look on.

### CODE REVIEW

File: ibis/expr/api.py [hunk 1]
```diff
@@ -152,6 +152,14 @@ def desc(expr):
 def timestamp(value):
     """
     Returns a timestamp literal if value is likely coercible to a timestamp
+
+    Parameters
+    ----------
+    value : timestamp value as string
+
+    Examples
```
Comment (cpcloud) [inline ibis/expr/api.py]: This should be `Returns`.

File: ibis/expr/api.py [hunk 2]
```diff
@@ -152,6 +152,14 @@ def desc(expr):
...
+    --------
+    result : Timestamp scalar
```
Comment (cpcloud) [inline ibis/expr/api.py]: This actually returns an `ibis.expr.types.TimestampScalar`, you can just write `TimestampScalar`.

File: ibis/expr/api.py (hunk 1 above)
Comment (djKooks) [inline ibis/expr/api.py]: Pretty dumb mistake...I fixed it.

File: ibis/expr/api.py (hunk 2 above)
Comment (djKooks) [inline ibis/expr/api.py]: @cpcloud Thanks! I updated it.

**Comment by cpcloud** [comment]
@djKooks Looks good. Thanks! Merging.

**Comment by djKooks** [comment]
@cpcloud Thanks!

**Pull request closed without merging**